talent/job-descriptions/culture-section.md

```diff
@@ -1,11 +1,13 @@
 ## Culture 
 
-- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the community, not necessarily companies. "For developers" is the answer to any strategic question or dilemma in the company. Always For Developers.
+- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the individual developer in the community. Our team consists of members who are passionate about programming. To understand our culture better, read more about it [here](https://github.com/src-d/guide/blob/master/general/culture.md#for-developers).
 
-- "By developers" is our identity. Everyone in the company is either a developer or has a developer/engineering mindset. We don't hire anyone who doesn't fit into this culture, it's the only way to make sure everyone understands developers and can always defend their interest.
+- At the moment, we are 25+ people from 10 different countries working as a distributed organization. Some of our team members are based in the Madrid, Spain or San Francisco office, others work remotely from around the world (Portugal, Estonia, Russia, and others).
 
-- At the moment, we are 20+ people from 10 different countries working closely together from our office in Madrid, Spain. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely as part of our team does working from Portugal, Estonia, Russia, and others.
+- For those wanting to work from one of our offices, we fully support the visa and moving process for you and your family.
 
-- At source{d}, we have a transparent salary policy which we feel strongly about it. Although a seniority level for every candidate will be determined during the last round of on-site interviews, one can expect that for a Senior Engineer salary ranges from €49.000 to €53.000. You can find our net amounts [here](https://cincodias.elpais.com/herramientas/calculadora-sueldo-neto/) (in Spanish only).
+- At source{d}, we have a transparent salary policy which we feel strongly about it. Your seniority level will be determined during the last round of on-site interviews.
 
-- Following the same level of transparency as with development process, all the above aspects of the work culture are documented and publicly available in our Guide at https://github.com/src-d/guide/.
\ No newline at end of file
+- At source{d} all of the projects we work on are public on GitHub and the vast majority are open-source under licenses such as Apache 2.0.
```

Comment:
In these documents I've read:
- Open Source
- open source
- open-source

Choose one and keep it consistent